pydoctor/astbuilder.py

```diff
@@ -52,33 +52,6 @@ def node2fullname(expr, ctx):
         return None
 
 
-def _get_function_signature_annotations(func_ast: ast.FunctionDef) -> Mapping[str, ast.expr]:
```

Comment:
This code seems to have been moved whole-sale into the method. This makes it harder to figure out if any of the logic changed. I'm not sure why it moved? Why couldn't the other function just call this as is?

Reply:
It was moved in commit 3ac25183, which doesn't change the logic. It only becomes hard to figure out when reviewing all changes at once...

In any case, the reason for the move is to be able to call `_unstring_annotation()`, which is a method of `ModuleVistor`.

The only thing `_unstring_annotation()` needs from `ModuleVisitor` is `self.builder.currentMod.report()`, so an alternative would be to pass the logger as an argument instead. But that is not the style in most of the pydoctor code is currently written.